playbook/app/pb_kits/playbook/pb_circle_chart/_circle_chart.tsx

```diff
@@ -75,6 +78,7 @@ const CircleChart = ({
   htmlOptions = {},
   id,
   innerSize = "md",
+  itemToMove,
```

Comment:
Why are we exposing this to the end user? 
If the story is just about getting the prop on the kit table, I wouldn't want us to add a new feature by creating the ability for a dev to customize this (by enabling this prop to them).

@markdoeswork

Reply:
I guess i missed that detail i've removed it